packages/amplify_authenticator/lib/src/blocs/auth/auth_bloc.dart

```diff
@@ -146,6 +146,26 @@ class StateMachineBloc {
     }
   }
 
+  Stream<AuthState> _isValidSession() async* {
+    try {
+      bool isValidSession = await _authService.isValidSession;
+      if (isValidSession) {
+        yield const Authenticated();
+      } else {
+        yield AuthFlow.signin;
+      }
+    } on AmplifyException catch (e) {
+      /// On all AmplifyExceptions other than SignedOutException or SessionExpired,
+      /// we provide a message to the user and surface the SignIn form.
+      _exceptionController.add(AuthenticatorException(e.message));
+      yield AuthFlow.signin;
+    } on Exception catch (e) {
+      /// On all other Exceptions we display an an exception and do not
+      /// load the Auththenticator, as the app may be in an unusable state
+      _exceptionController.add(AuthenticatorException(e.toString()));
```

Comment:
This results in the end user seeing a CircularProgressIndicator and a banner with the error message. If an end user actually encountered this, that isn't great. It might be better to just show the the sign in screen.

Reply:
I went back and forth on this, and felt that maybe not loading anything would be better. But... I agree that an infinite spinner is really maddening.  We could potentially red-screen the app, but I don't think that's much better. For now let's just show SignIn.